python/tk_desktop/loading_project_widget.py

```diff
@@ -107,14 +110,22 @@ def show_error_message(self, msg):
         :param str msg: Message to display
         """
         # Hide all widgets
-        self._ui.shotgun_spinning_widget.hide()
-        self._ui.bottom.hide()
-        self._ui.progress_output.hide()
+        self._show_hide_widgets(show=False)
 
         self.setVisible(True)
         self._message = msg
         self.repaint()
 
+    def _show_hide_widgets(self, show=False):
```

Comment:
I think this should just be called `_show_widgets()`. The boolean argument implies you're either showing or not showing I think. The calls above become:

`self._show_widgets(True)`

and 

`self._show_widgets(False)`